### DISCUSSION TITLE
How to change timestamp format for RFC message

### DISCUSSION THREAD

**Discussion by dbrosluc** [IonxSolutions/serilog-sinks-syslog#49] [General]
Hello,

Is there an easy way to change/configure timestamp format (i.e UTC) on RFC message?

Thank you

**Comment by colin-anderson** [comment]
I'm not really sure what you mean, both `Rfc3164Formatter` and `Rfc5424Formatter` generate timestamps that adhere to their respective RFC specifications? Do you mean you want to send messages with non-standard timestamps?

**Reply by dbrosluc** [reply]
Hi Colin, 

You remind me that we can use RFC5424Formatter to set the timestamp with MessageTemplateTextFormatter. I am able to change the timestamp format.

Thank you,
